typescript/infra/config/environments/testnet2/helloworld.ts

```diff
@@ -14,5 +14,7 @@ export const helloWorld: HelloWorldConfig<TestnetChains> = {
     chainsToSkip: [],
     runEnv: environment,
     namespace: environment,
+    prometheusPushGateway:
+      'http://prometheus-pushgateway.monitoring.svc.cluster.local:9091',
```

Comment:
It's not the same cluster. Both mainnet and testnet will point to the pushgateway that is in their cluster, they are just mirrored so it's not super obvious because the domain names end up being the same.